### DISCUSSION TITLE
Histogram Charts

### DISCUSSION THREAD

**Discussion by LaurinX** [qax-os/excelize#1186] [Q&A] [answered]
Hey there,
is there a way to create charts without using data from the worksheet?
I am trying to create a histogram using 2D clustered column chart since its the closest thing to a histogram. My raw data is displayed on the worksheet but i do not want to put the "processed" data (eg. intervals from my raw data for the categories field in formatChartSeries) on the worksheet. 
If any of you know a workaround or something please let me know

**Comment by xuri** [comment, marked as answer]
Creates a chart by `AddChart` with a data source that references another sheet and hides the data source sheet by [`SetSheetVisible`](https://pkg.go.dev/github.com/xuri/excelize/v2#File.SetSheetVisible). Another way is to create a chart sheet by [`AddChartSheet`](https://pkg.go.dev/github.com/xuri/excelize/v2#File.AddChartSheet) instead of the AddChart function.
